blog/2024-11-20-pants2-23/index.mdx

```diff
@@ -0,0 +1,58 @@
+---
+  authors: [benjy]
+  tags: [announcement]
+  # The release happened on this date, despite what the file name implies (which is when this post was authored)
+  date: 2024-11-19
+---
+
+# Pants 2.23.0 is released!
+
+<CaptionedImg src={require("./splash.jpeg").default}>
+  Photo by [Sarah
+  Kilian](https://unsplash.com/@rojekilian?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash)
+  on
+  [Unsplash](https://unsplash.com/photos/two-grey-walking-elephants-E-szZQqUmH4?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash)
+</CaptionedImg>
+
+{/* truncate */}
+
+We are pleased to announce Pants [2.23.0](<(https://github.com/pantsbuild/pants/tree/2.23.x/src/python/pants/notes/2.23.x.md)>), the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.23.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/2.23/docs/getting-started).
+
+The previous release, 2.22.0, suffered from delays in its release cycle. So this 2.23.0 release is a relatively fast follow, and does not introduce major upheavals.
+
+A couple of administrative reminders:
+
+- The annual [Pants user survey](https://www.pantsbuild.org/blog/2024/10/08/user-survey-2024) is open through the end of 2024. The results will be used to assist the Pants maintainers and contributors to better understand areas where Pants may need work, so your feedback is vital!
+- We're very proud that Pants now has its first [sponsors](https://www.pantsbuild.org/sponsors)! We are very grateful to [Klayvio](https://www.klaviyo.com/) and [Normal Computing](https://normalcomputing.ai/) for their Platinum tier support, and to [Continua](https://www.continua.ai/) and Stormfish for their Silver tier support.
```

Comment:
Should Stormfish get a link?

Reply:
They don't have a website yet. Am waiting on it.